src/XGBoostSharp/XGBClassifier.cs

```diff
@@ -266,4 +266,27 @@ public float[][] PredictProbability(DMatrix dMatrix)
 
         return results;
     }
+
+    public Array Predict(
```

Comment:
@madskonradsen The method seems similar to the `XGBoostRegressor` one, so might as well place the method on `XGBModelBase` to avoid duplication.

Reply:
Good point :)